deployment/helpers/deployment-helpers.js

```diff
@@ -16,12 +16,13 @@ async function deployCDKValidiumDeployer(deployerAddress, signer) {
     const gasPrice = ethers.BigNumber.from(ethers.utils.parseUnits(gasPriceKeylessDeployment, 'gwei'));
     const to = '0x'; // bc deployment transaction, "to" is "0x"
     const tx = {
-        to,
-        nonce: 0,
-        value: 0,
-        gasLimit: gasLimit.toHexString(),
-        gasPrice: gasPrice.toHexString(),
-        data: deployTxCDKValidiumDeployer,
+      to,
+      chainId: network.config.chainId ?? 0, // some RPCs only support replay-protected txs (EIP-155)
```

Comment:
Is this comment relevant to us null coalescing to zero?  If not, let's move it somewhere else and consider throwing an exception if chain id is null

Reply:
network.config.chainId!

Reply:
yes `network.config.chainId` can be null and it's equivalent to setting it to 0